component/cleanup-cronjob.libsonnet

```diff
@@ -0,0 +1,96 @@
+/*
+* Deploys the control-api API server
+*/
+local common = import 'common.libsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local inv = kap.inventory();
+local params = inv.parameters.control_api;
+local com = import 'lib/commodore.libjsonnet';
+
+local namespace = {
+  metadata+: {
+    namespace: params.namespace,
+  },
+};
+
+local hasCountriesConfig = params.odoo8.countries != null && std.length(params.odoo8.countries) > 0;
+
+local extraDeploymentArgs =
+  (if hasCountriesConfig then
+     [
+       '--billing-entity-odoo8-country-list=/config/billing_entity_odoo8_country_list.yaml',
+     ]
+   else
+     [])
+;
+
+local countryList = std.filterMap(
+  function(name) params.odoo8.countries[name] != null,
+  function(name) {
+    name: name,
+    code: params.odoo8.countries[name].code,
+    id: params.odoo8.countries[name].id,
+  },
+  std.objectFields(params.odoo8.countries)
+);
+
+local countriesConfigMap =
+  kube.ConfigMap('billing-entity-odoo8-country-list') {
+    metadata+: {
+      namespace: params.namespace,
+    },
+    data: {
+      'billing_entity_odoo8_country_list.yaml': std.manifestYamlDoc(countryList),
+    },
+  };
```

Comment:
As far as I can see, this is unnecessary copy-pasted logic, as we already create the exact same object in `component/api-server.jsonnet` and you don't even emit the `countriesConfigMap` object in this file.

```suggestion
```